crates/apollo-mcp-server/src/server/states/running.rs

```diff
@@ -273,6 +281,28 @@ impl ServerHandler for Running {
             health_check.record_rejection();
         }
 
+        let attributes = vec![
+            KeyValue::new(
+                "success",
+                result.is_ok()
+                    && result
+                        .as_ref()
+                        .ok()
+                        .map(|r| r.is_error != Some(true))
+                        .unwrap_or(false),
```

Comment:
I guess this can be a one liner?

```suggestion
                result.as_ref().is_ok_and(|r| r.is_error != Some(true)),
```

Reply:
Good catch, implemented and unit test passed